deploy/helm/sumologic/templates/_helpers/_common.tpl

```diff
@@ -608,5 +616,5 @@ Example usage:
 {{- end -}}
 
 {{- define "sumologic.sumologic-mock.name.roles.serviceaccount" -}}
-{{- template "sumologic.fullname" . }}
+{{- template "sumologic.fullname" . }}-sumologic-mock
```

Comment:
I think we already know this is a "Sumo Logic" mock. Also we have had issues with names being too long.
```suggestion
{{- template "sumologic.fullname" . }}-mock
```

Reply:
Oh I see the name `...-sumologic-mock` is already being used in `_sumologic_mock.tpl`, I'm apparently a bit late with this suggestion. Can we still change those names to be shorter?

Reply:
sure, I will prepare it as separate PR to merge before this one

Reply:
#3536